### PR TITLE
Angular-SDK-Transition

### PR DESCRIPTION
**Pull request by Erice59**
Transition to Angular SDK from webjs.

### CODE REVIEW

**Comment by Erice59** [comment]
@chris4490

**Review by chris4490 — COMMENTED** [review]
A few items to double check on.

**Review by allenzhou101 — CHANGES_REQUESTED** [review]
See slack comments

File: README.md
Comment (gaokevin1) [inline README.md:1]: Make sure you change this from saying it uses the web-component to using the Angular SDK, and link to the SDK as well.

File: package.json
Comment (gaokevin1) [inline package.json:1]: Run `npm audit fix` to update one of the dependencies.

File: src/app/auth.service.ts [hunk 1]
```diff
@@ -39,7 +39,7 @@ export class AuthService {
         return user;
       } else if (!sessionToken || this.sdk.isJwtExpired(sessionToken)) {
         try {
-          await this.sdk.refresh();
+          await this.sdk.refresh(this.sdk.getRefreshToken());
```
[the remark this answers is not in the document]
Comment (Erice59) [reply src/app/auth.service.ts:41]: Done.

File: README.md
Comment (Erice59) [reply README.md:1]: That should be reflected in the readme, unless it should be elsewhere too?

File: package.json
Comment (Erice59) [reply package.json:1]: We suggest yarn in the readme because of some weird dependency issues.

**Review by chris4490 — APPROVED** [review]

**Review by chris4490 — APPROVED** [review]

**Pull request merged**